service-base/src/main/java/io/narayana/lra/logging/LraI18nLogger.java

```diff
@@ -48,7 +48,7 @@ public interface LraI18nLogger {
     String error_noContentOnGetStatus(URI coordinator, URL lra);
 
     @Message(id = 25007, value = "LRA coordinator '%s' returned an invalid status for LRA '%s'")
-    String error_invalidArgumentOnStatusFromCoordinator(URI coordinator, URL lra, @Cause Throwable t);
+    String error_invalidArgumentOnStatusFromCoordinator(URI coordinator, URL lra);
```

Comment:
I don't normally like to change existing log messages, is removing the exception strictly necessary?

Reply:
This message is triggered by a client asking for the status of the LRA and the stack trace will only show client side logic but the actual failure has happened remotely at the coordinator where the real interesting state is so the stack isn't useful. But I'm not fussed if you prefer it being reinstated?

Reply:
I would prefer to keep the old log message intact if possible even though like you say it seems not so useful but maybe it could be some day and we made the call before to have the stack trace.

My point was really to avoid changing existing log messages, rather than the utility in practice - more exploring precedent in altering existing messages.

Reply:
I understand and concur :-)